fastlane/Fastfile

```diff
@@ -531,54 +537,16 @@ platform :ios do
 
     build_app_store_connect
 
-    testflight(
-      skip_waiting_for_build_processing: true,
-      team_id: TEAM_ID_APP_STORE,
-      api_key: app_store_connect_api_key
-    )
+    upload_app_store_connect_build_to_testflight
 
     symbols_upload
 
-    next unless create_release
-
-    archive_zip_path = File.join(PROJECT_ROOT_FOLDER, 'PocketCasts.xcarchive.zip')
-    zip(path: lane_context[SharedValues::XCODEBUILD_ARCHIVE], output_path: archive_zip_path)
-
-    app_version = release_version_current
-    build_version = build_code_current
-    version = beta_release ? build_version : app_version
-
-    UI.message("Creating #{version} release on GitHub...")
-    set_github_release(
-      repository_name: GHHELPER_REPO,
-      api_token: get_required_env!('GITHUB_TOKEN'),
-      name: version,
-      tag_name: version,
-      description: begin
-        File.read(File.join(RESOURCES_FOLDER, 'release_notes.txt'))
-      rescue StandardError
-        'No changelog provided'
-      end,
-      commitish: Git.open(PROJECT_ROOT_FOLDER).log.first.sha,
-      upload_assets: [archive_zip_path.to_s],
-      is_draft: !beta_release,
-      is_prerelease: beta_release
-    )
-
-    UI.message('Sending message to Slack...')
-    slack(
-      pretext: slack_message(version: app_version, build_number: build_version, is_beta: beta_release),
-      default_payloads: [],
-      slack_url: get_required_env!('SLACK_WEBHOOK'),
-      fail_on_error: false
-    )
-
-    FileUtils.rm_rf(archive_zip_path)
+    create_release_on_github(beta_release: beta_release) if create_release
   end
 
   desc 'Builds for distribution to App Store Connect'
-  lane :build_app_store_connect do
-    configure_code_signing_app_store
+  lane :build_app_store_connect do |fetch_code_signing: true|
+    configure_code_signing_app_store if fetch_code_signing
```

Comment:
It can be handy to skip the time consuming code signing fetching locally during development iterations.